lib/global_id/locator.rb

```diff
@@ -19,8 +19,11 @@ def locate(gid, options = {})
       end
 
       # Takes an array of GlobalIDs or strings that can be turned into a GlobalIDs.
-      # The GlobalIDs are located using Model.find(array_of_ids), so the models must respond to
-      # that finder signature.
+      # All GlobalIDs must belong to the same app, as they will be located using
+      # the same locator using its locate_many method.
+      #
+      # By default the GlobalIDs will be located using Model.find(array_of_ids), so the
```

Comment:
Global IDs encode the model_class and I'd love to keep the language consistent here (iirc we only say model class and not just model).

What about:
"Global IDs are located by calling find with an array of model ids on the model class."

I guess this means `find` should accept one id or many (which before locate_many it didn't).